x-pack/plugins/ml/public/application/services/ml_api_service/jobs.ts

```diff
@@ -48,6 +48,15 @@ export const jobsApiProvider = (httpService: HttpService) => ({
     });
   },
 
+  jobsForExport(jobIds: string[]) {
```

Comment:
See earlier comment - what's the thinking behind using `export` in this route rather than `cloning`? Will it ultimately be used for import / export of job configs?

Reply:
Updated to be `jobForCloning` here https://github.com/elastic/kibana/pull/88898/commits/37e73ae597cc4acd9db6db59a17ec893799d240d